core/admin/mailu/ui/views/users.py

```diff
@@ -93,12 +93,12 @@ def user_settings(user_email):
     form = forms.UserSettingsForm(obj=user)
     utils.formatCSVField(form.forward_destination)
     if form.validate_on_submit():
-        if form.forward_enabled.data and (form.forward_destination.data in ['', None] or type(form.forward_destination.data) is list):
+        user.forward_enabled = bool(flask.request.form.get('forward_enabled', False))
+        if form.forward_enabled and not form.forward_destination.data:
```

Comment:
I still think it should be 
```suggestion
        if user.forward_enabled and not form.forward_destination.data:
```
which was set to a bool. But I don't know if "form.forward_enabled" also returns the truth value of its data.

Reply:
That's 11e72e70